packages/init/main.tf

```diff
@@ -1,3 +1,60 @@
+
+# Enable Secrets Manager API
+resource "google_project_service" "secrets_manager_api" {
+  #project = var.gcp_project_id
```

Comment:
Is there a reason why to keep the project here?

Reply:
No, good call--had that in there for testing. Removing it